plugins/foreman_ansible/1.x/index.md

```diff
@@ -87,8 +89,22 @@ The project is not packaged, but you can find the sources here at [theforeman/fo
 
 In Foreman, you should add whatever Ansible hosts you want to submit facts from to the setting `trusted_puppetmaster_hosts`. Change it at Administer > Settings, Auth tab.
 
+Ansible will run using the SSH key of the smart-proxy or Foreman. These keys are automatically set by Foreman Remote Execution, so you may see them at /usr/share/foreman-proxy/.ssh . The foreman-proxy user needs write access to '/usr/share/foreman-proxy/.ansible' at least to create the '.ansible' local directory. We are working on making the instller automatically do this, but if it didn't do it.
```

Comment:
Can we move that to ```/etc/foreman-proxy/ssh``` and only install a symlink in ```/usr/share/foreman-proxy```?

Reply:
We can do that, but for 1.16 users this is not an option

Reply:
I don't like putting promises of a change in documentation. Either refer to a redmine issue or leave it out IMHO.